pkg/controllers/managementuser/snapshotbackpopulate/snapshotbackpopulate.go

```diff
@@ -254,6 +322,16 @@ func (h *handler) configMapToSnapshots(configMap *corev1.ConfigMap, cluster *pro
 				Controller:         &[]bool{true}[0],
 				BlockOwnerDeletion: &[]bool{true}[0],
 			}}
+			snapshot.Annotations[StorageAnnotationKey] = StorageS3
+		} else {
```

Comment:
So, this code is executed if the snapshot is local (stored on a node) right? I thought it'd be helpful to have a comment here to make sure that's clear.